CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+### Fixed
```

Comment:
Maybe we could also append the part about logging every stacktrace for every kind of unhandled exceptions?